integrations/database/postgresql/postgres.yaml

```diff
@@ -12,12 +12,6 @@ spec:
     # columns: connection-type database user address auth-options
     pg_hba:
       - hostnossl  all  all  all  reject
-  bootstrap:
```

Comment:
if we keep the kubearchive-user, should we keep these lines too?

Reply:
In case we keep it yes, but I don't think we need to keep it.